edgexfoundry/device-rfid-llrp-go
# refactor!: Use latest SDK for flattened config stem

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-rfid-llrp-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

- unit tests: make test
- integration testing:
1. Run EdgeX non-secure stack so Consul is running
2. Build the service locally make build
3. cd cmd
4. export EDGEX_SECURITY_SECRET_STORE=false; /device-rfid-llrp -cp and
    verify that the service configuration is at http://localhost:8500/ui/dc1/kv/edgex/v3/

## New Dependency Instructions (If applicable)
